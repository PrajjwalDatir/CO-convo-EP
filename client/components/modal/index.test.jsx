import React from 'react';
import { mount } from 'enzyme';
import Modal from './index';

describe('<Modal />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<Modal />);
    expect(wrapper).toMatchSnapshot();
  });
});
