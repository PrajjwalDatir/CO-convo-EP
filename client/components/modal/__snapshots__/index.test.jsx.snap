// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> should match snapshot 1`] = `
<Modal
  bgClassName="bg-gray-900 opacity-25"
  isOpen={false}
  onClose={[Function]}
>
  <PopupMenu
    bgClassName="bg-gray-900 opacity-25"
    isOpen={false}
    onClose={[Function]}
  >
    <div
      className="invisible"
    >
      <button
        aria-label="Close menu"
        className="fixed inset-0 w-full h-full cursor-default z-40 bg-gray-900 opacity-25"
        onClick={[Function]}
        tabIndex="-1"
        type="button"
      />
      <div
        className="relative z-50"
      >
        <div
          className="fixed w-screen h-screen inset-0 pointer-events-none"
        >
          <div
            className="flex w-screen h-screen items-center justify-center"
          >
            <div
              className="pointer-events-auto"
            />
          </div>
        </div>
      </div>
    </div>
  </PopupMenu>
</Modal>
`;
